AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Main_Null_Render_Component_02.py

```diff
@@ -4,13 +4,16 @@
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
+import sys
+
 import pytest
 
 from ly_test_tools.o3de.editor_test import EditorBatchedTest, EditorTestSuite
 
 
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 @pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.skipif(sys.platform.startswith("linux"), reason="https://github.com/o3de/o3de/issues/13930")
```

Comment:
in other suites we use `@pytest.mark.skipif(ly_test_tools.LINUX, reason="https://github.com/o3de/o3de/issues/13930")` along with it's `import ly_test_tools`

Reply:
oops, well im going to leave it as is for now but will remember to use this method instead in the future like in the other suites

Reply:
reason being is i dont want the pipeline to get blocked by one of these intermittent failures